### PR TITLE
[Doc] update getting started guide for emr 6.8.0 release[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
Signed-off-by: liyuan <yuali@nvidia.com>

udpate getting started guide for emr 6.8.0 release

### CODE REVIEW

**Comment by nvliyuan** [comment]
verified
image

File: docs/get-started/getting-started-aws-emr.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Different versions of EMR ship with different versions of Spark, RAPIDS Accelera
 
 | EMR | Spark | RAPIDS Accelerator jar | cuDF jar | xgboost4j-spark jar
 | --- | --- | --- | ---| --- |
+| 6.8 | 3.3.0 | rapids-4-spark_2.12-22.06.0.jar | X | xgboost4j-spark_3.0-1.4.2-0.3.0.jar |
```
Comment (sameerz) [inline docs/get-started/getting-started-aws-emr.md:17]:
```suggestion
| 6.8 | 3.3.0 | rapids-4-spark_2.12-22.06.0.jar | Bundled with rapids-4-spark | xgboost4j-spark_3.0-1.4.2-0.3.0.jar |
```

**Review by sameerz — APPROVED** [review]

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Comment by sameerz** [comment]
build

**Pull request merged**